script/Ciel/Day 14/14_01B_01_CIEL14B_1E.txt

```diff
@@ -1,12 +1,16 @@
 [sha:a333af1f407c16c92cfa62caab9e0801551769c8]{
 -- Page 10005, Offset 38970.
 -- 代行者が第七聖典による狙撃を敢行せんとする、数分前。
-A few minutes before the Executor attempted to shoot with the Seventh Holy Scripture against all odds.
+// The postfix notation on this line and next doesn't really work nicely in EN,
+// so rendered as a sort of formulaic prefix instead?
+Time: A few minutes before the Executor launches her desperate sniper attack with the Seventh Scripture.
 }
 [sha:27aa1c6dbdbd8f5ae261c3f06fd788b85fb560c0]{
 -- Page 10005, Offset 38971.
 -- 事象が歪みはじめた、その中心。
-At the center of the warping phenomenon.
+// Haven't rendered 事象 as 'events' here, maybe we should?
+// It's kinda confusing outside of the set phrases though
+Place: The epicenter of the distortion affecting reality.
```

Comment:
I mean the probability link isn't clear in EN at this point at all imo. <事|モ><象|ノ> is rendered as <Events|m a t t e r   >, and there's a bunch of garbage about 'flattening time' and '2d-ification' that make it seem way more like it's some form of compression than something to do with probabilities.

Also, we gotta capitalize Event if we're going to be using it here, because event is a very normal word.

Reply:
I feel like it shouldn't be needed, honestly, given the amount of description up until this point. If someone doesn't get it without the caps, I'm uncertain they would with it. I suppose it doesn't really hurt, but I do feel like it looks very ugly.